web/business-registry-dashboard/app/locales/fr-CA.ts

```diff
@@ -214,6 +215,14 @@ export default {
     noSubscription: {
       title: 'Création de Compte Incomplète',
       description: 'Terminez la création de votre compte. Assurez-vous d\'avoir ajouté le produit Business Registry à votre compte et réessayez votre lien.'
+    },
+    adminAccountSetupIncomplete: {
+      title: 'Création de Compte Incomplète',
```

Comment:
I think "Incomplète" should be masculin. Not sure.

Reply:
My internal grammar rules got piqued by this but "création" is indeed feminine so just leave this, I think.